openid/src/test/java/com/inrupt/client/openid/OpenIdSessionTest.java

```diff
@@ -157,7 +157,7 @@ void testClientCredentials() {
         assertFalse(session.fromCache(null).isPresent());
         final Optional<URI> principal = session.getPrincipal();
         assertEquals(Optional.of(URI.create(WEBID)), principal);
-        assertTrue(session.fromCache(null).isPresent());
+        assertFalse(session.fromCache(null).isPresent());
```

Comment:
Duplicates assertion from 3 lines above

Reply:
In the past, the cache would be populated at this point (which was incorrect). This assertion tests that the cache is still not populated.